conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -16,7 +16,6 @@ dependencies:
 - cuda-version=12.9
 - cudf==25.10.*,>=0.0.0a0
 - cupy>=13.6.0
-- cuvs==25.10.*,>=0.0.0a0
```

Comment:
If you remove this will setting up dev environments be impacted ?

Reply:
No, this is intentional. We have never had a dependency on `cuvs` python package, so this dependency was wrongly expressed. There should be no side-effects to removing this.